From: Michael Biebl <biebl@debian.org>
Date: Thu, 10 Jul 2014 03:33:18 +0200
Subject: Use symlinks for nmtui

Make /usr/bin/nmtui-* symbolic links to /usr/bin/nmtui.
---
 clients/tui/Makefile.am | 2 +-
 configure.ac            | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/clients/tui/Makefile.am b/clients/tui/Makefile.am
index ad9335d..09f2700 100644
--- a/clients/tui/Makefile.am
+++ b/clients/tui/Makefile.am
@@ -24,7 +24,7 @@ links = nmtui-edit nmtui-connect nmtui-hostname
 
 install-exec-hook:
 	for link in $(links); do \
-		ln -f $(DESTDIR)$(bindir)/nmtui $(DESTDIR)$(bindir)/$$link; \
+		cd $(DESTDIR)$(bindir) && $(LN_S) -f nmtui $$link; \
 	done
 
 uninstall-hook:
diff --git a/configure.ac b/configure.ac
index c21cb9d..ae2ec6d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -45,6 +45,8 @@ AM_PROG_CC_C_O
 # C++ only required if --enable-qt=yes
 AC_PROG_CXX
 
+AC_PROG_LN_S
+
 dnl Initialize libtool
 LT_PREREQ([2.2])
 LT_INIT([disable-static])
